cli/azd/cmd/init.go

```diff
@@ -280,7 +280,7 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 
 		header = "Your app is ready for the cloud!"
 		followUp = "Run " + output.WithHighLightFormat("azd up") + " to provision and deploy your app to Azure.\n" +
-			"Run " + output.WithHighLightFormat("azd infra synth") + " to write infrastructure as code files to disk " +
+			"Run " + output.WithHighLightFormat("azd infra gen") + " to write infrastructure as code files to disk " +
```

Comment:
```suggestion
			"Run " + output.WithHighLightFormat("azd infra gen") + " to generate IaC for your project to disk, allowing you to manually manage it. \n" +
```

Keeping it consistent to IaC, which is what we use in other space. Also keeps the line shorter